CONTRIBUTING.md

```diff
@@ -6,11 +6,16 @@ We really want Parse to be yours, to see it grow and thrive in the open source c
 
 ##### Please Do's
 
-* Take testing seriously! Aim to increase the test coverage with every pull request.
-* Run the tests for the file you are working on with `npm test spec/MyFile.spec.js`
-* Run the tests for the whole project and look at the coverage report to make sure your tests are exhaustive by running `npm test` and looking at (project-root)/lcov-report/parse-server/FileUnderTest.js.html
-* Lint your code by running `npm run lint` to make sure all your code is not gonna be rejected by the CI.
-* Never publish the lib folder.
+* Begin by reading the [Development Guide](http://docs.parseplatform.org/parse-server/guide/#development-guide) to learn how to get started running the parse-server.
+* Take testing seriously! Aim to increase the test coverage with every pull request. To obtain the test coverage of the project, run:
+    * **Windows**: `npm run coverage:win`
```

Comment:
I'm wondering if we should consolidate these so it's just `npm run coverage` rather than one or the other for platform issues. We could keep `npm run coverage:win` and just make it an alias for the former.

Reply:
problem is the env variables that are not processed properly on windows :/

Reply:
We could modify `npm run coverage` to os detect and run from there? Would be a bit more code underlying, but I don't believe we'd ever need to worry about which command to tell someone to use in the future and we probably wouldn't ever change it.

Reply:
I would agree that consolidating into one would be great. It's unfortunate that npm doesn't support having scripts for specific OSes but oh well. 

I'll give a shot at consolidating this into one script.

Reply:
(addressing both :win scripts)
`test:win` wouldn't be required if all `test` did was test - this is what `test:win` does - suggest
* Remove `test` and `coverage`
* Rename `test:win` to `test` - this will only perform testing
* Rename `coverage:win` to `coverage` and ensure works on Unix/Mac by using the correct path for jasmine that works for both platforms

Can probably shorten to
`cross-env MONGODB_VERSION=${MONGODB_VERSION:=3.2.6} MONGODB_STORAGE_ENGINE=mmapv1 TESTING=1 node nyc ./node_modules/jasmine/bin/jasmine.js`

Reply:
Actually here's what we use in another project for coverage (on Windows):
`cross-env NODE_ENV=test TESTING=1 nyc jasmine`